### PR TITLE
New runtime

### PR DESCRIPTION
**Pull request by bbhtt**

### CODE REVIEW

File: com.github.mgropp.PdfJumbler.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "id": "com.github.mgropp.PdfJumbler",
     "runtime": "org.freedesktop.Platform",
-    "runtime-version": "20.08",
+    "runtime-version": "22.08",
```
Comment (hfiguiere) [inline com.github.mgropp.PdfJumbler.json:4]: why not 23.08?

Comment (bbhtt) [reply com.github.mgropp.PdfJumbler.json:4]: Openjdk11 doesn't have a branch yet

**Comment by bbhtt** [comment]
Runs

**Comment by nbenitez** [comment]
Thanks 👍🏻

**Pull request merged**